website/docs/cloud-docs/vcs/azure-devops-services-pat.mdx

```diff
@@ -1,28 +1,34 @@
 ---
-page_title: Set up the Azure DevOps Services VCS provider using Personal Access Tokens
+page_title: Set up the Azure DevOps Services VCS provider using personal access tokens
 description: >-
-  Learn how to use Azure DevOps Services with workspaces and private registry modules in HCP Terraform using Personal Access Tokens.
+  Learn how to use Azure DevOps Services with workspaces and private registry modules in HCP Terraform using personal access tokens.
 ---
 
-# Set up the Azure DevOps Services VCS provider using Personal Access Tokens
+# Set up the Azure DevOps Services VCS provider using personal access tokens
 
-These instructions are for using `dev.azure.com` for HCP Terraform's VCS features. [Other supported VCS providers](/terraform/cloud-docs/vcs) have separate instructions.
+These instructions describe how to use a personal access token to connect HCP Terraform to Azure DevOps Services VCS, which is hosted at `dev.azure.com`. Connecting to Azure DevOps Services VCS lets you take advantage of HCP Terraform's VCS features. 
+
+If you do not intent to use personal access tokens to connect to Azure DevOps Services, refer to [Set up the Azure DevOps Server VCS provider](/terraform/cloud-docs/vcs/azure-devops-server). For information about other supported VCS providers, refer to [Connect to VCS Providers](/terraform/cloud-docs/vcs).
 
 ## Requirements
 
 Configuring a new VCS provider requires permission to [manage VCS settings](/terraform/cloud-docs/users-teams-organizations/permissions#manage-vcs-settings) for the organization.
 
+The account you use for connecting HCP Terraform must have **Project Collection Administrator** access enabled for any projects containing repositories of Terraform configurations. This permission is required to create webhooks. You cannot create custom access roles with lower levels of privilege because Microsoft does not allow delegation of this capability.
+
 [permissions-citation]: #intentionally-unused---keep-for-maintainers
 
-## Important Notes About Authentication
+## Personal access token maintenance
 
 HCP Terraform uses personal access tokens to connect to Azure DevOps Services. This access method requires some additional configuration and ongoing maintenance:
 
-- Personal access tokens eventually expire, with a maximum allowed lifetime of one year. If HCP Terraform's token expires, it will be unable to connect to Azure DevOps Services until the token is replaced. To avoid a gap in service, do one of the following before the token expires:
-  - Update the expiration date of the existing token within Azure DevOps Services.
-  - Create a new token, and edit HCP Terraform's VCS connection to use it.
+- Personal access tokens have a maximum lifetime of one year. When the token expires, HCP Terraform cannot to connect to Azure DevOps Services until the token is replaced. To avoid a gap in service, do one of the following actions before the token expires:
```

Comment:
```suggestion
- Personal access tokens have a maximum lifetime of one year. When the token expires, HCP Terraform cannot connect to Azure DevOps Services until the token is replaced. To avoid a gap in service, do one of the following actions before the token expires:
```